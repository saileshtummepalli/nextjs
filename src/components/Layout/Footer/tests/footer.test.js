
import Footer from "../footer";
import {render, shallow,getRenderOutput} from "enzyme"

describe('<Footer />', () => {
  it('Should shallow correctly', () => {
    const  container  = shallow(<Footer />);
    expect(container.debug()).toMatchSnapshot();
  });
  describe('<Footer />', () => {
    it('enzyme shallow', () => {
      const  container  = shallow(<Footer />);
      const wapper = container.find(`[data-test='footerComponent']`);
      expect(wapper.length).toBe(1);
    });
  })
});