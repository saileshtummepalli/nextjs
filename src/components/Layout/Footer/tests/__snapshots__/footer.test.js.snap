// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Should render correctly 1`] = `
"<div data-test=\\"footerComponent\\">
  <div className=\\"container mx-auto px-6\\">
    <div className=\\"mt-16 border-t-2 border-gray-300 flex flex-col items-center\\">
      <div className=\\"sm:w-2/3 text-center py-6\\">
        <p className=\\"text-sm text-blue-700 font-bold mb-2\\">
          Â© 2021 by poc
        </p>
      </div>
    </div>
  </div>
</div>"
`;
